### PR TITLE
PA: Create new standalone download page for /software url

### PR DESCRIPTION
**Pull request by moollaza**
Requires internal changes.

### CODE REVIEW

File: share/site/duckduckgo/base.tx [hunk 1]
```diff
@@ -13,42 +13,48 @@
...
-	  <: if !$no_header { include "header.tx" } :>
-	  <div id="content_wrapper" class="content-wrap">
-	  <: if !$no_content_internal { :>
-		<div id="content" class="content<: if !$no_cw { :>  cw<: } :>">
-			<div id="content_internal" class="content__internal">
-	  <: } :>
-		<: } else {:>
-		  <: if $homepage { :>
+		<: if !$no_header { :>
+			<: if $transparent_header { :>
```
Comment (bbraithwaite) [inline share/site/duckduckgo/base.tx]: Is this being used anywhere?

File: share/site/duckduckgo/base.tx [hunk 2]
```diff
@@ -13,42 +13,48 @@
...
+				<: include "header.tx" :>
+			<: } else if $flex_header { :>
+				<: include "header_flex.tx" :>
+			<: } else {:>
+				<: include "header.tx" :>
+			<: } :>
+		<: } :>
+		<div id="content_wrapper" class="content-wrap">
```
Comment (bbraithwaite) [inline share/site/duckduckgo/base.tx]: This is just a formatting change right? Let me know if there is anything I should be looking at here ?

File: share/site/duckduckgo/base.tx (hunk 1 above)
Comment (moollaza) [reply share/site/duckduckgo/base.tx]: Not that I could see. Can probably remove!

File: share/site/duckduckgo/base.tx (hunk 2 above)
Comment (moollaza) [reply share/site/duckduckgo/base.tx]: Yup the only new thing is the check for `$flex_header` -- I may not be able to keep this though as I realized now it breaks the header for the old (currently live) design. I may need to go back to using CSS to create this UI for now.

```perl
<: } else if $flex_header { :>
    <: include "header_flex.tx" :>
```

If you use the 'diff settings' above to hide whitespace changes, the new content becomes much more clear:

duckduckgo-publisher 2018-06-19 10-35-40

**Pull request merged**